mmdet/core/evaluation/eval_hooks.py

```diff
@@ -54,6 +54,10 @@ def __init__(self,
                  save_best=None,
                  rule=None,
                  **eval_kwargs):
+        warnings.warn(
```

Comment:
We should let this module inherits from the EvalHook supported in MMCV, and only override the `__init__` function by adding the warnings. The other functions can use those in MMCV.